core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/ToBytesViaStringDeserialiser.java

```diff
@@ -47,6 +47,19 @@ public void setCharset(final String charset) {
         this.charset = charset;
     }
 
+    /**
+     * Deserialise an array of bytes into the original object.
+     *
+     * @param bytes the bytes to deserialise
+     * @return T the deserialised object
+     * @throws SerialisationException if the object fails to deserialise
+     * <p>
+     * Note that this implementation is less efficient than using deserialise
+     * with an offset and a length, but may still be used if necessary.
+     * It has been marked deprecated but will not be removed as it is needed
+     * in the Serialiser interface.
+     * @see #deserialise(byte[], int, int)
+     */
     @Deprecated
     @Override
     public final T deserialise(final byte[] bytes) throws SerialisationException {
```

Comment:
This function, as well as [ToBytesSerialiser.deserialise(byte[])](https://github.com/gchq/Gaffer/blob/ab0b66c11f99d30d6c9b9f004e09f6166e206188/core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/ToBytesSerialiser.java#L85), have been left in despite being deprecated. This is because they cannot be removed as they need to have implementations for the Serialiser [interface](https://github.com/gchq/Gaffer/blob/ab0b66c11f99d30d6c9b9f004e09f6166e206188/core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/Serialiser.java#L73). However, the deprecated tag has been left in to warn people to instead use the more efficient deserialise(byte[], int, int) method instead.